src/Search.css

```diff
@@ -0,0 +1,13 @@
+
+.App-header {
```

Comment:
avoid using capitals for anything other than js classes or components. use lower case with dashes for CSS classes